multisense_ros/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(multisense_ros)
 
-# Default to C++14
+# Default to C++17
 if(NOT CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD 17)
```

Comment:
Seems a safe change. @QuentinTorg?

Reply:
This might break the ROS driver for older compilers without 17 support. I wonder if we could only do this for GCC version greater than 11